Free Wraith Clones With Low HP

Lukas-bit opened the pull request:
Clones that attempted to heal the main wraith if their health was <=25% max hp were not being freed. This caused wraiths with low hp to heal the main wraith every 15-30 seconds until they were killed, instead of dying after they healed. The overall boss fight should hopefully be more fair now.

Lukas-bit committed:
Merge branch 'master' into lukas/transient_wraith

Lukas-bit committed:
Merge branch 'master' of https://github.com/noahwagner04/CSE320_Game

Lukas-bit committed:
Merge branch 'master' of https://github.com/noahwagner04/CSE320_Game

Lukas-bit committed:
Merge branch 'master' of https://github.com/noahwagner04/CSE320_Game

Lukas-bit committed:
Merge branch 'master' of https://github.com/noahwagner04/CSE320_Game

Lukas-bit committed:
Merge branch 'master' of https://github.com/noahwagner04/CSE320_Game

Lukas-bit committed:
Merge branch 'master' into lukas/transient_wraith

Lukas-bit committed:
Free Wraiths With Low Health Instead of Splitting

Clones that were going heal the main wraith if their health was at 25% or below
max hp were not being freed. This caused wraiths with
low hp to just infinitely heal the main wraith every 15-30 seconds.
The overall boss fight should hopefully now be more fair.

Lukas-bit committed:
Merge branch 'master' of https://github.com/noahwagner04/CSE320_Game

Lukas-bit committed:
Merge branch 'master' into lukas/transient_wraith

Lukas-bit committed:
Merge branch 'lukas/transient_wraith' of https://github.com/noahwagner04/CSE320_Game into lukas/transient_wraith

noahwagner04 reviewed (approved):
looks good! you can merge

Lukas-bit committed:
Increase Behemoth Vole Projectile Damage

Increased the damage that the projectile will do on contact, as well
as the overall poison damage
